script.js

```diff
@@ -0,0 +1,33 @@
+$(document).ready(function(){
```

Comment:
^^ this works! but is not required because script tag is just before closing body.